core/src/main/resources/jenkins/security/apitoken/ApiTokenPropertyConfiguration/help-creationOfLegacyTokenEnabled.html

```diff
@@ -0,0 +1,5 @@
+<div>
+    Even if this is disabled (recommended for security reasons), the users are still able to generate new tokens
```

Comment:
nit: I would reword this blurb to: "This option allows users to generate a legacy API token if they do not already have one. Because legacy tokens are \<strong\>deprecated\</strong\>, we recommend disabling it and having users instead generate new API tokens from the user configuration page."